cleanlab_studio/studio/studio.py

```diff
@@ -144,3 +144,70 @@ def apply_corrections(self, cleanset_id: str, dataset: Any) -> Any:
             corrected_ds[label_column] = joined_ds["__cleanlab_final_label"]
             corrected_ds = corrected_ds[joined_ds["action"] != "exclude"]
             return corrected_ds
+
+    def create_project(
+        self,
+        dataset_id: str,
+        project_name: str,
+        modality: str,
+        *,
+        tasktype: str = "multi-class",
+        modeltype: str = "regular",
```

Comment:
suggestion: let's type these as literals instead of strings, to make clearer to the user what the choices are